Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -82,6 +92,11 @@ class DashboardViewModel {
                     }
                 }
                 self.events = events
+                self.loadingState = .loaded
+            } else {
+                showFailedLoadErrorAlert.toggle()
```

Comment:
note because of what I mentioned earlier, use the existing variables as opposed to the new ones